### PR TITLE
[ADD] stock_move_value_report: New module

### PR DESCRIPTION
**Pull request by chienandalu**
This module allows to obtain a report with the cost value of a given
operation set like a picking, a scrap, an inventory or even a selection of
moves or move lines.

cc @Tecnativa TT18619

### CODE REVIEW

File: i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,172 @@
+# Translation of Odoo Server.
```
Comment (pedrobaeza) [inline i18n/es.po]: Incorrect path

Comment (chienandalu) [reply i18n/es.po]: Oops. Fixed :)

File: stock_move_value_report/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock Move Cost Value Report",
+    "version": "11.0.1.0.0",
+    "category": "Stock",
```
Comment (rousseldenis) [inline stock_move_value_report/__manifest__.py]: Set it to 'Warehouse' as it won't appear in good category on OCA shop.

File: stock_move_value_report/report/stock_move_line_value_report.xml [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+                        Lot/Serial Number
+                    </th>
+                    <th name="th_qty" class="text-center">
+                        <strong>Quantity</strong>
+                    </th>
+                    <th name="th_value" class="text-right">
+                        <strong>Value</strong>
+                    </th>
+                    <th name="th_value_sbtl" class="text-right">
+                        <strong>Value Subtotal</strong>
```
Comment (rousseldenis) [inline stock_move_value_report/report/stock_move_line_value_report.xml]: IMHO 'Subtotal' should be sufficeint

File: stock_move_value_report/report/stock_move_line_value_report.xml [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+                        </t>
+                    </t>
+                </div>
+                <!-- Total all moves -->
+                <div class="clearfix">
+                    <div id="total" class="row">
+                        <div class="col-xs-4 pull-right">
+                            <table class="table table-condensed" style="min-width: 250px;max-width: 400px;page-break-inside: avoid;">
+                                <tr class="border-black">
+                                    <td><h4>Total operations value balance</h4></td>
```
Comment (rousseldenis) [inline stock_move_value_report/report/stock_move_line_value_report.xml]: This one is too big related to the 'Total' line.

image

File: stock_move_value_report/report/stock_move_value_report.xml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="report_stock_move_value">
+        <t t-call="stock_move_value_report.report_stock_move_value_document">
+            <t t-set="move_lines_ids" t-value="docs.mapped('move_line_ids')"/>
```
Comment (rousseldenis) [inline stock_move_value_report/report/stock_move_value_report.xml:6]: This works only if 'Show detailed operations' is checked on picking type. Either the report is void.

File: stock_move_value_report/report/stock_picking_value_report.xml [hunk 6]
```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="report_stock_picking_value_document">
+        <t t-call="web.html_container">
+            <t t-call="web.external_layout">
+                <div class="page">
+                    <div class="row">
+                        <div class="col-xs-6">
+                            <div t-if="picking.move_lines and picking.move_lines[0].partner_id and picking.move_lines[0].partner_id.id != picking.partner_id.id">
```
Comment (rousseldenis) [inline stock_move_value_report/report/stock_picking_value_report.xml:10]: As above, this works only if you checked the detailed operations.

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code and functional review.

Missing the case if move_lines are not used by config.

File: stock_move_value_report/report/stock_move_value_report.xml (hunk 5 above)
Comment (chienandalu) [reply stock_move_value_report/report/stock_move_value_report.xml:6]: No, it isn't. The stock.move.line is created as soon as move is reserved or done even if you don't see it. Indeed what you see in the stock picking delivery slip is the move lines created.

File: stock_move_value_report/report/stock_picking_value_report.xml (hunk 6 above)
Comment (chienandalu) [reply stock_move_value_report/report/stock_picking_value_report.xml:10]: Same here.

File: stock_move_value_report/report/stock_move_line_value_report.xml (hunk 3 above)
Comment (chienandalu) [reply stock_move_value_report/report/stock_move_line_value_report.xml]: Well, in cases like AVCO, the customer wants to see the price unit of the operation so it's clear at what cost the operation was made

File: stock_move_value_report/report/stock_move_value_report.xml (hunk 5 above)
Comment (rousseldenis) [reply stock_move_value_report/report/stock_move_value_report.xml:6]: Of course, my bad. Didn't check the state of picking.

File: stock_move_value_report/report/stock_move_line_value_report.xml (hunk 4 above)
Comment (chienandalu) [reply stock_move_value_report/report/stock_move_line_value_report.xml]: Well, it's to make a distinction with the partials coming from pickings, inventories or scraps from the move lines selected

File: stock_move_value_report/report/stock_move_line_value_report.xml (hunk 3 above)
Comment (rousseldenis) [reply stock_move_value_report/report/stock_move_line_value_report.xml]: It was just for the string title. On the report, it's quite big.

File: stock_move_value_report/report/stock_move_line_value_report.xml (hunk 4 above)
Comment (rousseldenis) [reply stock_move_value_report/report/stock_move_line_value_report.xml]: Yes, but it's a little bit ugly, no ?

**Review by rafaelbn — APPROVED** [review]
Tested in runbot :+1: 

@rousseldenis is it ok for you?

File: stock_move_value_report/report/stock_move_line_value_report.xml (hunk 4 above)
Comment (chienandalu) [reply stock_move_value_report/report/stock_move_line_value_report.xml]: Ok, I changed the layout a bit to fit in a more integrated way.

**Review by rousseldenis — APPROVED** [review]
Great!

**Pull request merged**